_posts/2020-01-20-pulp-3-GA-demos.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Pulp 3.0 GA Demos
+author: Dana Walker
+tags:
+  - 3.0
+---
+To go along with Pulp 3.0 GA, we have prepared a playlist of demos to cover features in pulpcore as
+well as various features from some of our plugins.
+
+We will continue adding content to this playlist, so be sure to check it out and let us know if you
```

Comment:
I think the playlist covers all the features of Pulpcore so I don't think we'll continue adding to it.